src/components/BibPage/BibDetail.tsx

```diff
@@ -13,14 +21,53 @@ const buildDetailElement = (field: BibDetail) => {
   return (
     <>
       <dt>{field.label}</dt>
-      {field.value.map((val: string, i: number) => {
-        const stringDirection = displayRtl(val)
-        return (
-          <dd dir={stringDirection} key={i}>
-            {val}
-          </dd>
-        )
-      })}
+      <dd>
+        <List data-testId={`${field.label}`} type="ol" noStyling>
+          {field.value.map((val: string, i: number) => {
+            const stringDirection = displayRtl(val)
+            return (
+              <li dir={stringDirection} key={i}>
+                {val}
+              </li>
+            )
+          })}
+        </List>
+      </dd>
+    </>
+  )
+}
+
+const buildSingleSubjectHeadingElement = (subjectHeadingUrls: Url[]) => {
+  const urls = subjectHeadingUrls.reduce((linksPerSubject, url: Url, index) => {
+    // Push a divider in between the link elements
+    const divider = (
+      <span data-testid="divider" key={`divider-${index}`}>
+        {" "}
+        &gt;{" "}
+      </span>
+    )
+    const link = linkElement(url, "internal")
+    linksPerSubject.push(link)
+    if (index < subjectHeadingUrls.length - 1) linksPerSubject.push(divider)
+    return linksPerSubject
+  }, [] as React.JSX.Element[])
+  return urls
+}
+
+const buildCompoundSubjectHeadingElement = (field: SubjectHeadingDetail) => {
+  const subjectHeadingLinksPerSubject = field.value.map(
+    (subjectHeadingUrls: Url[]) => {
+      return buildSingleSubjectHeadingElement(subjectHeadingUrls)
+    }
+  )
+  return (
+    <>
+      <dt>{field.label}</dt>
+      {subjectHeadingLinksPerSubject.map((subject, i) => (
+        <Box data-testid="subjectLinksPer" key={"subject-" + i}>
```

Comment:
This can be a `dd`. As a `Box` (which renders a `div`), the term won't have a definition associated with it.

Reply:
that makes sense. i didn't like it as a box anyhow.